content/css/concepts/display/terms/clear/clear.md

```diff
@@ -42,3 +42,5 @@ Making an `img` element move down a _left_ floated `h1` element.
   clear: left;
 }
 ```
+
```

Comment:
Add a sentence introducing the image and explaining it is an example of the above CSS.